docs/workflows/best-practices-osh.md

```diff
@@ -0,0 +1,398 @@
+  | CVSSv3 Range | Label | Remediation Deadline Critical | Remediation Deadline Regular |
+  | --- | --- | --- | --- |
+  | 9.0 – 10.0 | Critical | Within 1 working day | Within 1 working day | 
+  | 7.0 – 8.9 | High | Within 14 days | Within 30 days |
+  | 4.0 – 6.9 | Medium | Within 60 days | Within 90 days |
+  | 0.1 – 3.9 | Low | Within 1 year | Within 1 year |
+  > TODO: check that the above table makes sense, especially the added last column
+
+- If no remediation is available, do a risk assessment which will have one of 3 outcomes:
+  - If we find that the vulnerability does not pose any actual risk, we will ‘allowlist’ it. This requires CISO approval. This allowlist will be reviewed as part of the half-yearly measurement cycle.
+  - We will mitigate the risk in some other way. If, for example, we do not want to allowlist. the entire library because of its importance but the vulnerability is limited to a single method, we can test for the use of that method and fail the pipeline in that case.
+  - [In](http://3.in/) extreme cases, we will shut down the application until the vulnerability is resolved.
+  - note: when addressing the vulnerabilities of a specific library by updating to a newer version, that always also improves the freshness
+  
+
+### 6. Handling detected license issues [ROUGH DRAFT]
+- Libraries must have an acceptable license.
+  - This may be a paid license or an acceptable open-source license. [Company] maintains a list of common licenses used in free and open-source software (FOSS); if a library has a license listed as acceptable, it can be used. Otherwise, see if an alternative is available, or contact the [applicable role] to discuss whether the license is acceptable.
+- See also the License Risk Evaluation Framework slide at the bottom of the wiki page: [OSH interpret results wiki page: https://softwareimprovementgroup.atlassian.net/wiki/spaces/SSM/pages/50317951024/How+to+interpret+Open-Source+health+findings](https://workflowy.com/#/c9c4745b6b6b)
+- Note that the risk depends on the context: 
+  - e.g. when developing open-source software, more licenses are acceptable.
+  - it also depends on how you use a library: it is sometimes needed/a solution to wrap the library in an executable component that can be used by calling, instead of becoming a part of the codebase.
+
+
+### 7. Handling detected lack of freshness [ROUGH DRAFT]
+- The teams are responsible for keeping libraries reasonably up to date: 
+  - small library updates can be updated as part of regular maintenance; 
+  - larger updates (e.g. major versions or frameworks) should be planned explicitly as part of a development sprint.
+
+### 8. _Handling detected lack of activity_ (TBD)
+
+## Handling your libraries
+
+
+### 9. Updating a library [ROUGH DRAFT]
+There can be several reasons to consider updating a library.
+
+Checklist before updating:
+- Does it have open issues?​
```

Comment:
Not sure what this check is meant to cover. A well-maintained library [will have open issues.](https://github.com/spring-projects/spring-boot/issues). Zero open issues typically equates to low activity. I would say 'Check the release notes to see if and how you might be affected by the changes.'.